Source/MLXNN/Activations.swift

```diff
@@ -468,9 +476,15 @@ open class ReLU6: Module, UnaryLayer {
 ///
 /// ### See Also
 /// - <doc:activations>
-open class SoftMax: Module, UnaryLayer {
+open class Softmax: Module, UnaryLayer {
+    public var axis: Int
+
+    public init(axis: Int = -1) {
+        self.axis = axis
+    }
```

Comment:
GLU already had this (exposed axis parameter) and the original request was to add negative slope.  This just exposes axis on Softmax and LogSoftmax.